modules/programs/bash.nix

```diff
@@ -123,6 +123,15 @@ in
           interactive shell.
         '';
       };
+
+      logoutExtra = mkOption {
```

Comment:
I would prefer `logoutExtraConfig` to remain consistant.

Reply:
I named it `logoutExtra` to remain consistant with `initExtra`, `bashExtra` etc